pfcpiface/main.go

```diff
@@ -122,15 +123,24 @@ func ParseIP(name string, iface string) net.IP {
 }
 
 func main() {
-	log.SetFlags(log.LstdFlags | log.Lshortfile)
-
 	// cmdline args
 	flag.Parse()
 	var conf Conf
 	var intf fastPath
 	// read and parse json startup file
 	ParseJSON(configPath, &conf)
-	log.Println(conf)
+
+	// Set up logger
+	log.SetReportCaller(true)
+	log.SetFormatter(&log.TextFormatter{
+		FullTimestamp: true,
+	})
+	if level, err := log.ParseLevel(conf.LogLevel); err != nil {
+		log.Fatalln(err)
```

Comment:
if `conf.LogLevel` is nil, set it to info and then do this. Currently if I do not have `log_level` in my config, its exiting.

```
FATA[2021-08-07T07:36:23Z]/pfcpiface/main.go:139 main.main() not a valid logrus Level: ""
```

Reply:
I added that check to `ParseJSON`

Reply:
CI is passing now. That must have been the cause for its failure 😀